mtenn/readout.py

```diff
@@ -81,3 +79,46 @@ def forward(self, delta_g):
             pic50 -= torch.log10(torch.tensor(self.cp_val, dtype=delta_g.dtype))
 
         return pic50
+
+
+
+class KiReadout(Readout):
+    """
+    Readout implementation to convert delta G values to Ki values. This new
+    implementation assumes implicit energy units, WHICH WILL INVALIDATE MODELS TRAINED
+    PRIOR TO v0.3.0.
+    Assuming implicit energy units:
+        deltaG = ln(Ki)
+        Ki = exp(deltaG)
+    """
+
+    def __init__(self):
+        """
+        Initialization.
+
+        Parameters
+        ----------
+        None
+        """
+        super(KiReadout, self).__init__()  #TODO Remove?
+
+    def __repr__(self):
+        return f"KiReadout()"
+
+    def forward(self, delta_g):
+        """
+        Method to convert a predicted delta G value into a Ki value.
+
+        Parameters
+        ----------
+        delta_g : torch.Tensor
+            Input delta G value.
+
+        Returns
+        -------
+        float
+            Calculated Ki value.
+        """
+        ki = torch.exp(delta_g).item()
```

Comment:
```suggestion
        ki = torch.exp(delta_g)
```
We actually do want to return the Tensor so that we keep the pytorch computation graph intact. I'm realizing now that I have the docstring wrong in the `PIC50Readout` class, but if  you look at the code, we pass through the `delta_g` Tensor the whole time. if you could change both of these docstrings actually that would be appreciated, just `float` -> `torch.Tensor`